misc/inmanta-workon-register.sh

```diff
@@ -144,6 +144,13 @@ function __inmanta_workon_list {
 
 
 function __inmanta_workon_activate {
+    # Check user
+    current_user = $(whoami)
+    if [ [ $current_user != "root" ] && [ $current_user != ${INMANTA_USER:-inmanta} ] ]; then
+        echo "WARNING: The inmanta-workon tool should be run as either root or the inmanta user to have write access (to be able to run pip install or inmanta project install)." >&2
```

Comment:
Since we actually return here without doing anything I would make this an error.
```suggestion
        echo "ERROR: The inmanta-workon tool should be run as either root or the inmanta user to have write access (to be able to run pip install or inmanta project install)." >&2
```

Reply:
That or don't abort in this case. I think aborting is fine though. @wouterdb wdyt?

Reply:
I would not abort and warn